streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -3030,7 +3030,7 @@ private StreamTask createSingleSourceStateless(final StreamsConfig config,
             topology,
             consumer,
             new TopologyConfig(null,  config, new Properties()).getTaskConfig(),
-            new StreamsMetricsImpl(metrics, "test", builtInMetricsVersion, time),
```

Comment:
Can we remove `builtInMetricsVersion` as parameter of `createSingleSourceStateless(...)` too? Seems it's unused now?

Reply:
Sure! Thanks for this catch, Matthias!